app/data/etl/ga.rb

```diff
@@ -27,6 +28,15 @@ def extract_events
     end
   end
 
+  def transform_events
+    events_with_prefix = Events::GA.where("page_path ~ '^\/https:\/\/www.gov.uk'")
```

Comment:
Could you please extract a method for the prefix transformation. This way we don't care about the implementation details when reading the code. Something similar to: `fix_invalid_prefix_in_page_paths`?